lib/logflare/sources.ex

```diff
@@ -507,6 +507,7 @@ defmodule Logflare.Sources do
   def preload_for_dashboard(sources) do
     sources
     |> Enum.map(&preload_defaults/1)
+    |> Enum.map(&preload_rules/1)
```

Comment:
can consider getitng rid of this as we don't really need it for dashboard usage, except in the relevant rules editing pages (source rules/backend rules)